pipeline/simulate.bash

```diff
@@ -29,16 +25,17 @@ log_error() {
 }
 
 usage() {
-  log_error "Usage: ${0} <environment name> [<extra docker args>...]"
+  log_error "Usage: ${0} <tfe organization> <environment name> [<extra docker args>...]"
 }
 
-if [ $# -lt 1 ]; then
+if [ $# -lt 2 ]; then
   usage
   exit 1
 fi
 
-ENVIRONMENT_NAME=${1}
-shift
+TFE_ORGANIZATION=${1}
+ENVIRONMENT_NAME=${2}
+shift 2
```

Comment:
Exoscale key and secret due to their secret nature. :D And the other two since they are normally passed as environment variables as well, I usually have them set in my shell.

I wouldn't mind making them all env vars though if you think that would be nicer.